container/container.go

```diff
@@ -1106,7 +1106,7 @@ func (b *ContainerWorker) ResourcesCreate(agreementId string, agreementProtocol
 			}
 
 			woutAliases := &docker.EndpointConfig{
```

Comment:
It would be good to rename `woutAliases` to something else, since there are aliases now

Reply:
I agree